content/_data/changelogs/weekly.yml

```
@@ -9391,6 +9391,94 @@
   # pull: 5029 (PR title: Queue (mainly) style consistency)
   # pull: 5033 (PR title: JDK 11.0.9 / 8.0.272+ compatible FunctionsTest#printThrowable())
 
+- version: '2.266'
+  date: 2020-11-10
+  changes:
+  - type: major bug
+    category: major bug
+    pull: 5046
+    issue: 64040
+    authors:
+    - timja
+    message: |-
+      Fix hidden page elements from radio blocks appearing when they should not.
+  - type: rfe
+    category: rfe
+    pull: 5008
+    issue: 62446
+    authors:
+    - aHenryJard
+    message: |-
+      Make the Maven Configuration of Global Tools compatible with configuration as code plugin.
+  - type: rfe
+    category: rfe
+    pull: 4294
+    issue: 41891
+    authors:
+    - jsoref
+    message: |-
+      Minor English cleanup for resource root url cleanup.
+  - type: major rfe
+    category: developer
+    pull: 4944
+    issue: 19561
+    authors:
+    - jglick
+    message: |-
+      Developer: Jenkins now uses an updated version of the XStream serialization library without custom patches.
+  - type: major rfe
+    category: developer
+    pull: 4848
+    authors:
+    - jglick
+    message: |-
+      Developer: Jenkins now uses Spring Security rather than its predecessor, Acegi Security.
+      Other bundled Spring libraries are also updated.
+  - type: rfe
```

Is it also major @fqueiruga ? 

```suggestion
  - type: major rfe
```

---

Made it a major in 884344495b